packages/@tinacms/cli/src/cmds/audit/validations/validateRichText.ts

```diff
@@ -0,0 +1,34 @@
+/**
+Copyright 2021 Forestry.io Holdings, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { AuditIssue, AuditWarning } from '../issue'
+
+export const validateRichText = (node) => {
+  let issues: AuditIssue[] = []
+  Object.keys(node._values)
```

Comment:
I don't think this will work for errors in nested `rich-text` fields.

Reply:
Updated to search through nested fields